Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.AllSatisfy.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions.Execution;
+using FluentAssertions.Specs.Equivalency;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Collections
+{
+    /// <content>
+    /// The AllSatisfy specs.
+    /// </content>
+    public partial class CollectionAssertionSpecs
```

Comment:
🔧 I think it would be better to use a non-partial class and replace the region with a nested class.

Reply:
`DataEquivalencySpecs` as referenced in the discussion is using a partial class:

```csharp
public partial class DataEquivalencySpecs
{
    public class DataRowEquivalencySpecs {}
}
```
and

```csharp
public partial class DataEquivalencySpecs
{
    public class TypedDataTableEquivalencySpecs : DataEquivalencySpecs {}
}
```
I've updated it locally to the following, is this what you are looking for?
```csharp
public partial class CollectionAssertionSpecs
{
    public class AllSatisfyCollectionSpecs {}
}
```

Reply:
Ah yes, it _has_ to be a `partial` class. My bad. But almost. Instead of `AllSatisfyCollectionSpecs`, just call it `AllSatisfy`. 

Sorry for the confusion.